src/resolve.jl

```diff
@@ -105,26 +105,28 @@ end
 _convert(::Type{T}, @nospecialize(x)) where {T} = convert(T, x)::T
 
 # For each (V, B) in this list, if Julia has libstdc++ loaded at version at least V, then
-# B is a compatible bound for libstdcxx_ng.
+# B is a compatible bound for libstdcxx-ng.
 # See https://gcc.gnu.org/onlinedocs/libstdc++/manual/abi.html.
+# See https://github.com/gcc-mirror/gcc/blob/master/libstdc%2B%2B-v3/config/abi/pre/gnu.ver
 # See https://gcc.gnu.org/develop.html#timeline.
-# Last updated: 2024-11-08
+# Last updated: 2025-05-11
 const _compatible_libstdcxx_ng_versions = [
-    (v"3.4.33", ">=3.4,<14.3"),
-    (v"3.4.32", ">=3.4,<14.0"),
-    (v"3.4.31", ">=3.4,<13.2"),
-    (v"3.4.30", ">=3.4,<13.0"),
-    (v"3.4.29", ">=3.4,<12.0"),
-    (v"3.4.28", ">=3.4,<11.0"),
-    (v"3.4.27", ">=3.4,<9.3"),
-    (v"3.4.26", ">=3.4,<9.2"),
-    (v"3.4.25", ">=3.4,<9.0"),
-    (v"3.4.24", ">=3.4,<8.0"),
-    (v"3.4.23", ">=3.4,<7.2"),
-    (v"3.4.22", ">=3.4,<7.0"),
-    (v"3.4.21", ">=3.4,<6.0"),
-    (v"3.4.20", ">=3.4,<5.0"),
-    (v"3.4.19", ">=3.4,<4.9"),
+    (v"3.4.34", ">=15.1,<16.1"),  # NOTE: hypothetical upper bound
+    (v"3.4.33", ">=14.1,<15.1"),
+    (v"3.4.32", ">=13.2,<14.1"),
+    (v"3.4.31", ">=13.1,<13.2"),
+    (v"3.4.30", ">=12.1,<13.1"),
+    (v"3.4.29", ">=11.1,<12.1"),
+    (v"3.4.28", ">=9.3,<11.1"),
+    (v"3.4.27", ">=9.2,<9.3"),
+    (v"3.4.26", ">=9.1,<9.2"),
+    (v"3.4.25", ">=8.1,<9.1"),
+    (v"3.4.24", ">=7.2,<8.1"),
+    (v"3.4.23", ">=7.1,<7.2"),
+    (v"3.4.22", ">=6.1,<7.1"),
+    (v"3.4.21", ">=5.1,<6.1"),
+    (v"3.4.20", ">=4.9,<5.1"),
+    (v"3.4.19", ">=4.8.3,<4.9"),
```

Comment:
Why have you tightened the ranges?

Reply:
Hum, I think you are right, this narrowing is too restrictive.
I'm restoring the previous lower bound now ...